modules/portworx/scripts/remove-portworx.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -u
+set -o pipefail
+. vars
+
+vPXStrgClstrName="px-cluster";
+"${kbCtl}"  --kubeconfig="${vKubeConfig}" patch   storagecluster "${vPXStrgClstrName}" --namespace portworx -p '{"spec":{"deleteStrategy":{"type":"UninstallAndWipe"}}}' --type=merge
+"${kbCtl}"  --kubeconfig="${vKubeConfig}" delete  StorageCluster "${vPXStrgClstrName}" --namespace portworx &
+sleep 10
+echo "Wating for portworx cleanup proces to finish. It will take about 5 minutes..."
+while kubectl --kubeconfig="${vKubeConfig}" get pods -n portworx | grep -i running| grep -v operator 2>&1 >/dev/null
```

Comment:
Why not look into StorageCluster instead? That is the one being removed and which ensures that all dependencies are deleted. This while loop can be replaced with

`kubectl wait --for=delete StorageCluster/px-cluster -n portworx --timeout=600s`

In any case, I tried with both approach and it seems it hangs here. I see pods/deploy/rs are deleted but the StorageCluster resource stays in `DeleteInProgress`

Reply:
Added kubectl command with 300s timeout. For now, the bug which we've encountered is being worked by portworx.
If in case we've hit the bug, module wont hang, it will timeout.